ALUM/ALUM/Components/OutlinedButton.swift

```diff
@@ -0,0 +1,60 @@
+//
+// Outlined Button View.swift
```

Comment:
Let's rename this file to OutlinedButtonStyle

Reply:
Can we do this as well? With the change of button height from Infinity to fixed

Reply:
Just realised, you added a OutlinedButtonStyle. You missed deleting this OutlinedButton.swift though so let's remove that.